server/server.js

```diff
@@ -13,4 +23,17 @@ app.get('/api/careers', (request, response) => {
   });
 });
 
+app.get('/add-career', (request, response) => {
+  response.sendFile('form.html', { root: __dirname + '/../public/' });
+});
+
+app.post('/send-career', (request, response) => {
+  addCareer(request.body, (err, res) => {
+    if (err) {
+      console.log(err);
```

Comment:
logging the error? should we send something back to the user if it wasn't successful